tools/js-tools/git-hooks/pre-push-hook.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-console, no-process-exit */
```

Comment:
`no-process-exit` is not needed anymore.
```suggestion
/* eslint-disable no-console */
```